lib/page/house_download.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Page
+  class HouseDownload
+    attr_reader :country, :house
+
+    def initialize(country:, house:, index:)
+      @country = country
+      @house   = house
+      @index   = index
+    end
+
+    def title
+      "EveryPolitician: #{country.name} - #{house.name}"
+    end
+
+    def legislative_periods
+      house.legislative_periods
+    end
```

Comment:
All the tests pass if I change this implementation to `house.legislative_periods.take(1)`, which probably isn't so good…